Makefile

```
@@ -2,7 +2,7 @@ help:
 	@echo "lint - check code style with flake8"
 	@echo "test - run tests only"
 	@echo "coverage - run tests and check code coverage"
-	@echo "conda_install (recommended) - Install requirements "
+	@echo "install - Install requirements "
 
 test:
 	py.test
```

```suggestion
	pytest
```
`py.test` is deprecated (see https://github.com/pytest-dev/pytest/issues/1629).